WhiteDentalClinic/WhiteDentalClinic.Application/Services/CustomerService.cs

```diff
@@ -21,27 +21,27 @@ public CustomerService(ICustomerRepository customerRepository, IMapper mapper, I
             _mapper = mapper;
             _claimService=claimService;
         }
-        public IEnumerable<CustomerResponseModel> GetAllCustomers()
+        public IEnumerable<ResponseCustomerModel> GetAll()
         {
             var responseModelListCustomers = _customerRepository.GetAll();
```

Comment:
I think just `customers` would be a better name. This one is quite long and the part that you are actually interested about is the last one "Customers" - the other 3 words you need to read until you get to "Customers" are just noise.